rapidJson-meson.patch

```diff
@@ -0,0 +1,39 @@
+From 5927bed25854f61eafac144837e2bfba60c721e3 Mon Sep 17 00:00:00 2001
+From: Dorinda Bassey <dbassey@redhat.com>
+Date: Wed, 26 Jan 2022 22:43:41 +0100
+Subject: [PATCH] Add RapidJSON dependency in Meson BuildSystem
+
+this RapidJSON dependency added in the Meson Build is needed for loading
+of Linux perf JSON. see reference - https://github.com/mikesart/gpuvis/pull/64
+
+Signed-off-by: Dorinda Bassey <dbassey@redhat.com>
+---
+ meson.build       | 5 +++++
+ meson_options.txt | 1 +
+ 2 files changed, 6 insertions(+)
+
+diff --git a/meson.build b/meson.build
+index 5652677..fc6d9e8 100644
+--- a/meson.build
++++ b/meson.build
+@@ -38,6 +38,11 @@ if get_option('use_i915_perf')
+   all_deps += dependency('i915-perf')
+ endif
+ 
++if get_option('have_rapidjson')
```

Comment:
Ideally this should be submitted upstream

Reply:
yes, this patch is upstream